index.html

```diff
@@ -243,10 +243,29 @@ <h2 class="text-center">Get in touch with us</h2>
         </section>
     </main>
     
-    <footer>
-        <div class="footer-text">Developed and Designed by <a href="https://github.com/sonianb"
+    <footer class="footer-container center">
+                
+                <img class="agency-logo center" src="img/VED-logo.png" alt="The Ved Agency Logo">
+                <p class="center footer-sizing">VED Agency, 2022</p><br>
+            
+            <div class="center footer-sizing">
+
+                <p>Tel: 0207 444 5629</p>
+                <p>email: VED@googlemail.com</p>
```

Comment:
I'd change that to hello@ved.com (doesn't @ved.com sound more professional?😄 )

Reply:
Thank you for the imput, it does sound better 😆 

:star: